libcontainer/process_linux.go

```diff
@@ -19,6 +19,9 @@ import (
 	"github.com/opencontainers/runc/libcontainer/utils"
 )
 
+// Synchronisation value for cgroup namespace setup.
+const privateCgroupns int = (1 << 7)
```

Comment:
I'm a bit mixed on this. On the one hand I think this should be implemented with an `#import` using `cgo` because I don't really want to duplicate this (especially since it's not really possible to `grep` for both usages of this constant). But then `cgo` is bad to use for something like that. :wink:

If we have to leave it here then please name it something similar to `SYNC_PRIVATE_CGNS` (after you rename that as per my comments above). Then at least it will be possible to `grep` for this constant.